### PR TITLE
🚧 test_BoardConverter.py test_when_double_convertation()

### PR DESCRIPTION
**Pull request by Serg-Bartosh**

### CODE REVIEW

File: tests/test_BoardConverter.py [hunk 1]
```diff
@@ -21,5 +21,32 @@ def test_convert_to_string(self):
...
+
+    def test_when_double_convertation(self):
+        board_srt = '0 0 0 0 0\n' \
+                    '0 1 0 1 0\n' \
+                    '0 0 0 0 0\n' \
+                    '1 0 0 0 1\n' \
+                    '0 1 1 1 0\n' \
+                    '0 0 0 0 0\n'
+
+        board_from_str_bool = BoardConverter.from_string(board_srt)
```
Comment (alena-bartosh) [inline tests/test_BoardConverter.py]:
```suggestion
        board_from_str = BoardConverter.from_string(board_srt)
```

File: tests/test_BoardConverter.py [hunk 2]
```diff
@@ -21,5 +21,32 @@ def test_convert_to_string(self):
...
+        board_from_str_bool = BoardConverter.to_string(board_from_str_bool)
```
Comment (alena-bartosh) [inline tests/test_BoardConverter.py]:
```suggestion
        str_from_board = BoardConverter.to_string(board_from_str_bool)
```

File: tests/test_BoardConverter.py [hunk 3]
```diff
@@ -21,5 +21,32 @@ def test_convert_to_string(self):
...
+
+        self.assertEqual(board_srt, board_from_str_bool)
```
Comment (alena-bartosh) [inline tests/test_BoardConverter.py]:
```suggestion
        self.assertEqual(board_srt, str_from_board)
```

**Review by s373r — APPROVED** [review]
Good

**Review by alena-bartosh — APPROVED** [review]
You passed this test :)

**Pull request merged**